spec/controllers/api/schedules/schedules_index_spec.rb

```diff
@@ -10,7 +10,7 @@
     it 'lists all schedules for a user' do
       sign_in user
       schedules = FactoryGirl
-                    .create_list(:schedule, 2, user: user)
+                    .create_list(:schedule, 2, device: user.device)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.